src/components/emptyStatePages/EmptyStatePage.tsx

```diff
@@ -192,19 +216,23 @@ export default function EmptyStatePage({ pageName, organization, reloadData }: E
         </>
       )}
       <PageHeader title={content.title1} subtitle='' />
-      <Container className={classes.mainContainer}>
-        <div className={classes.content}>
-          <EmptyStateContent
-            title={content.title2}
-            subtitle={content.subtitle}
-            listItems={content.listItems}
-            buttonText={content.buttonText}
-            buttonIcon={content.buttonIcon}
-            onClickButton={goToNewLocation}
-            styles={EMPTY_STATE_CONTENT_STYLES}
-          />
-        </div>
-      </Container>
+      {content.listItems.length === 0 ? (
+        <EmptyMessage className={classes.message} title={content.title2} text={content.subtitle} />
```

Comment:
empty state for species is instantiated in App.tsx
most other uses of EmptyMessage are within the component where title etc is already displayed.
added this functionality so we can support a contributor view for an emptystatepage

Reply:
Oh, thanks for clarifying this!